helm_inspect/utils/drift_check.py

```
@@ -0,0 +1,543 @@
+    "Deployment;template.spec.restartPolicy",
+    "Deployment;strategy.type",
+    "Service;clusterIP",
+    "Service;internalTrafficPolicy",
+    "Deployment;template.spec.containers[0].terminationMessagePolicy",
+    "Deployment;strategy.rollingUpdate.maxSurge",
+    "Deployment;template.spec.containers[0].resources",
+    "Deployment;template.spec.containers[0].ports[0].protocol",
+]
+
+
+from typing import Any, Dict
```

Ruff (0.8.2)

Redefinition of unused `Any` from line 11

Remove definition: `Any`

(F811)